plugins/filebrowser.c

```diff
@@ -686,6 +688,7 @@ static GtkWidget *create_popup_menu(void)
 	gtk_container_add(GTK_CONTAINER(menu), item);
 
 	item = gtk_check_menu_item_new_with_mnemonic(_("Show _Hidden Files"));
+	gtk_check_menu_item_set_active(GTK_CHECK_MENU_ITEM(item),show_hidden_files);
```

Comment:
Minor nitpick, but the space after the comma is missing.

Reply:
Sorry, a commit incoming.